tests/sparseml/pytorch/optim/test_modifier_quantization.py

```diff
@@ -63,7 +63,7 @@ def _is_valid_submodule(module_name, submodule_names):
     )
```

Comment:
let's add `num_calibration_steps` to `test_quantization_modifier_yaml`

Reply:
Added!